src/commons/api/admin.rs

```diff
@@ -598,6 +615,9 @@ impl fmt::Display for UpdateChildRequest {
         if let Some(resources) = &self.resources {
             write!(f, "new resources: {} ", resources)?;
         }
+        if let Some(suspend) = self.suspend {
+            write!(f, "change suspend status to: {}", suspend)?;
```

Comment:
Is it definitely a change? If it was Some before and is Some now that's now a change right?